c3.opam

```diff
@@ -1,6 +1,6 @@
 opam-version: "1.2"
 maintainer:   "dave@recoil.org"
-authors:      ["Dave Scott"]
+authors:      ["Dave Scott", "Thomas Gazagnaire"]
```

Comment:
I'm not sure a comma is valid syntax here

Reply:
oops sorry